src/Dialog/Dialog.js

```diff
@@ -184,6 +184,7 @@ module.exports = kind(
 	*/
 	titleChanged: function () {
 		var title = this.getTitle();
+		this.$.title.set('accessibilityLabel', title);
 		this.$.title.setContent( this.get('uppercase') ? util.toUpperCase(title) : title );
 	},
 
```

Comment:
I'll modify like below:
{from: 'title', method: function () {
	if (!this.accessibilityLabel) {
		var content = this.getTitle();
		this.$.title.setAriaAttribute('aria-label', content);
	}
}}